src/fhir/CarePlan.DCW.CogLevel1.NoDementiaDx.json

```diff
@@ -0,0 +1,75 @@
+{
+    "resourceType": "CarePlan",
+    "id": "TBD",
+    "description": "DCW cognition domain, level 1 (which is only for patients who don't have a dementia Dx)",
+    "status": "active",
+    "intent": "order",
+    "category": [
+        {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "719091000000102",
+                    "display": "Questionnaire"
+                }
+            ],
+            "text": "Questionnaire"
+        }
+    ],
+    "subject": {
+        "reference": "Patient/TBD",
+        "display": "Doe, John TBD"
+    },
+    "activity": [
+        {
+            "detail": {
+                "instantiatesCanonical": [
+                    "Questionnaire/[ID for ecog12]"
+                ],
+                "code": {
+                    "coding": [
+                        {
+                            "system": "http://snomed.info/sct",
+                            "code": "719091000000102",
+                            "display": "Questionnaire"
+                        }
+                    ],
```

Comment:
Will either remove this, or replace w/ a valid external reference.

Reply:
ok, right now nothing is returned if I search by [category](https://launch.smarthealthit.org/v/r4/fhir/CarePlan?subject=Patient/2cda5aad-e409-4070-9a15-e1c35c46ed5a&category=questionnaire) from smart health IT for a patient whose [careplan](http://launch.smarthealthit.org/v/r4/fhir/CarePlan?subject=Patient/2cda5aad-e409-4070-9a15-e1c35c46ed5a) I posted earlier today.

Reply:
@mcjustin this works for me when I modified the search parameter:  https://launch.smarthealthit.org/v/r4/fhir/CarePlan?subject=Patient/2cda5aad-e409-4070-9a15-e1c35c46ed5a&category:text=questionnaire

Reply:
@achen2401  please ignore this code block for now, there's nothing for you to action on here. But perhaps I'm misreading what you're saying - is this specific code blocking you from POSTing?

Reply:
thanks @mcjustin I will ignore this block for now.  Sorry, I guess I was trying to see, instead of loading all the carePlan for the patient (`/CarePlan?subject=[patientId]`), we could search for **questionnaire** only care plan (`/CarePlan?subject=Patient/2cda5aad-e409-4070-9a15-e1c35c46ed5a&category:text=questionnaire`)  (note, I added the additional search parameter, `category:text=questionnaire`).